torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -956,8 +924,7 @@ XLAGraphExecutor::PostOrderData XLAGraphExecutor::RunPostOrder(
 }
 
 XLAGraphExecutor::ComputationCache::TypePtr
-XLAGraphExecutor::LookupCachedCompile(const std::vector<XLATensorPtr>& tensors,
```

Comment:
haha, I guess at some point we dropped the use of `tensors` without noticing it.